airmozilla/popcorn/templates/popcorn/editor.html

```diff
@@ -23,6 +23,7 @@
 {% endblock %}
 
 {% block full_width_content %}
+<slug data-url="{{ slug }}"></slug>
 <div id="editor"></div>
```

Comment:
Wouldn't it be a bit more semantic to have the slug be a part of the whole page though? It isn't necessarily specific to the editor element.